tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -200,8 +200,15 @@ def send_slack_msg(self, key, message_payload):
                 response.status_code, response.text)
             return
 
-        response_json = response.json()
-        if not response_json['ok']:
-            self.logger.info("Error in sending Slack message. Status:%s, response:%s",
-                             response.status_code, response_json['error'])
-            return
+        if response.headers['content-type'] == 'text/html':
```

Comment:
I was just trying this with a wrong slack hook and 2 things I noticed: 

1. `Content-Type` instead of `content-type`.. Can we make it case-insensitive?
2. In my case header was actually `text/html; charset=utf8`.. Should we replace `==` with `contains`?

Reply:
1. This should be addressed by the _requests_ itself https://github.com/psf/requests/blob/master/requests/structures.py#L15
2. You right. Let me look at it closely.

Reply:
Content-Type header check has been adjusted according to the specification
https://github.com/slobodyanyuk/cloud-custodian/commit/252a30f839dd45527c4b5afc9616cabeeac45796